### ISSUE TITLE
Transfer Zeus to an github org

### ISSUE THREAD

**Issue by sideshowcoder** [burke/zeus#623]
I think do to the size it would maybe be a nice move to move zeus to an github org if possible, @burke what do you think?

**Comment by burke** [comment]
Makes sense to me. I see that github.com/zeus isn't available, but I have no objection to this if someone wants to go for it.

**Comment by sideshowcoder** [comment]
Mh yeah `github.com/zeus` would have been nice how about `zeus-preloader`? Doesn't roll of the tongue but described pretty well what it does. I don't really mind where the repo lives to be hones but feel like would be nice to be able to admin the repo for hooks and stuff. I'll take a look what would be needed to move and come back.

**Comment by pboling** [comment]
`zues-rails` ?

**Comment by burke** [comment]
would have been cool but it's been 9 years so
